### PR TITLE
docs: how to use UI features

### LINKED ISSUE #4927: [v2 DOCS] How to use UI features
Short guide on the basics of using the UI to annotate. Similar to the following content:
- Feedback dataset section here: https://docs.argilla.io/en/latest/practical_guides/annotate_dataset.html#feedback-dataset
- Quick mention to UI filters as well: https://docs.argilla.io/en/latest/practical_guides/annotate_dataset.html#feedback-dataset

Title could be "How to annotate records" (just a proposal).

**Notes:**
- The focus of the guide should be on the use of UI features to annotate.
- Some examples of features that should be mentioned: filters, sorting, similarity search, shortcuts, bulk, etc...
- Use as few screenshots as possible.  If something can be described with words, avoid adding a screenshot. These tend to get outdated fast and make the docs more difficult to maintain.

### PR DESCRIPTION
**Pull request by sdiazlor**
# Description

- How to guide
- Fix popular issues gh token requirement
- Fix links and reference links due to the renaming

Closes #4927 

**Type of change**

(Remember to title the PR according to the type of change)

- [x] Documentation update

**How Has This Been Tested**

(Please describe the tests that you ran to verify your changes.)

`mkdocs serve`

### CODE REVIEW

File: argilla/docs/how_to_guides/annotate.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+    If you want further guidance on good practices for guidelines during the project development, check our [blog post](https://argilla.io/blog/annotation-guidelines-practices/).
+
+## See the progress
+
+!!! info "Main use cases"
+    You want to see the progress of the annotation task.
+
+### General progress view
+
+On the main page, before accessing the dataset, you can see the general progress of the annotation task. This is indicated in the `Global progress` column that shows the number of remaining records to be annotated and a progress bar. The progress bar displays the percentage and number of records submitted, conflicting (i.e. those with overlap), discarded and pending by hovering your mouse over it.
```
Comment (sdiazlor) [inline argilla/docs/how_to_guides/annotate.md:187]: Maybe this is a bit picky, but is there a reason why it used "left" and not "pending" aligned with the rest of the record status?

File: argilla/docs/how_to_guides/annotate.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+description: In this section, we will provide a step-by-step guide to show how to annotate records in the UI.
+---
+
+# Annotate records
+
+This guide provides a detailed overview of how to annotate records in the UI, with separate workflows for each use case.
+
+To improve clarity, we will reference three main areas in the UI: the *preferences view* on the top left, the *record view* below in a white box, and the *annotation view* on the right.
+
+!!! tip
```
Comment (davidberenstein1957) [inline argilla/docs/how_to_guides/annotate.md:11]: perfect!

File: argilla/docs/how_to_guides/annotate.md [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+    * **Span**: Select pieces of text in the field linked to the span question and apply a label.
+    * **Text**: Provide a text response inside the text area.
+
+At the bottom of the annotation view, you can find the `Discard`, `Save as draft` and `Submit` buttons. The `Discard` button will discard the record if you decide not to provide any response. `Save as draft` will allow you to save your responses for review before submission. Finally, the `Submit` button will submit your responses for the given record, but it will be disabled until you provide responses for all required questions.
+
+### Navigation buttons
+
+To navigate through the records, you can use the `Prev`, shown as `<`, and `Next`, `>` buttons in the bottom right corner of the preferences view.
+
+### Shortcuts use
```
Comment (davidberenstein1957) [inline argilla/docs/how_to_guides/annotate.md]:
```suggestion
### Shortcut usage
```

File: argilla/docs/how_to_guides/annotate.md [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+    | Submit                                 | `↵ Enter`                         |
+    | Move between pages                     | `→ Right arrow` or `← Left arrow` |
+
+## Annotate in bulk
+
+!!! info "Main use cases"
+    * You have a good understanding of your data and a clear idea of the patterns to expect.
+    * You are doing basic text classification.
+
+!!! note
```
Comment (davidberenstein1957) [inline argilla/docs/how_to_guides/annotate.md]:
```suggestion
!!! note "Bulk is not available for SpanQuestions"
```

File: argilla/docs/how_to_guides/annotate.md [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+    The *bulk view* is not available for **span questions**. You will need to switch to the *focus view* for their annotation. In that case, we recommend you to annotate the rest of the questions in bulk, save them as drafts and answer the span questions in the *focus view* of the *Draft* queue.
```
Comment (davidberenstein1957) [inline argilla/docs/how_to_guides/annotate.md]:
```suggestion
    The *bulk view* is not available for **span questions**. You will need to switch to the *focus view* for their annotation. In that case, we recommend you annotate the rest of the questions in bulk, save them as drafts and answer the span questions in the *focus view* of the *Draft* queue. 
```

File: argilla/docs/how_to_guides/annotate.md [hunk 6]
```diff
@@ -0,0 +1,191 @@
...
+
+In the annotation view, on the right, you can find the questions to annotate. The annotation view in the bulk view is similar to the focus view. Each question has a title and the description (`i` icon), if available, and you will need to answer all the required questions marked with <span style="color:red; font-weight:bold;">*</span> to be able to submit the responses. However, suggestions are not available in the bulk view, so you will have to switch to the focus view if you need them.
+
+At the bottom of the annotation view, you can find the `Discard`, `Save as draft` and `Submit` buttons that will be enabled when at least one record is selected and, in the case of `Submit`, when all the required questions are annotated. To select specific records, you can click on the checkbox on the top left of each record. To select all records, you can click the checkbox in the bottom left of the preferences view. When discarded/saved as draft/submitted, **all the selected records** will change their status, so you can continue annotating the rest of the records.
+
+!!! Tip
```
Comment (davidberenstein1957) [inline argilla/docs/how_to_guides/annotate.md]: I think I might just add this at the top, to situations where to use it.

File: argilla/docs/how_to_guides/annotate.md [hunk 7]
```diff
@@ -0,0 +1,191 @@
...
+
+=== "Collapse records"
+
+    You can collapse the records to focus on the fields you are annotating. To do so, you can click on the `Record size` button at the bottom right of the preferences view. A dropdown will appear with the options `Expand records` and `Collapse records`.
+
+=== "Manage the number of records by page"
+
+    You can manage the number of records displayed by page. To do so, you can click on the `Page size` button at the bottom right of the preferences view. A dropdown will appear with the options `10`, `25`, `50` and `100`.
+
+## Understand patterns in your data
```
Comment (davidberenstein1957) [inline argilla/docs/how_to_guides/annotate.md]: I think this title is a bit vague and feels very passive
```suggestion
## Explore, find and understand patterns in your data 
```

File: argilla/docs/how_to_guides/annotate.md [hunk 8]
```diff
@@ -0,0 +1,191 @@
...
+    * You have numerous and/or long fields.
+    * You are familiar with your data.
+
+The UI provides different features to help you understand patterns in your data and manage the annotation task more efficiently.
+
+### Search and highlight
+
+You can search for records with specific words in their field content. For this, you should click on the magnifying glass icon in the top right of the preferences view. A search bar will appear where you can type the word or words you are looking for. If your records have multiple fields, you will be able to select a specific field for the search or select `All` to search on all the fields. Pressing `Enter` will show the records that contain these words, highlighted in <span style="color:#ff675f">coral</span>, in the selected fields.
+
+### Order by similar record
```
Comment (davidberenstein1957) [inline argilla/docs/how_to_guides/annotate.md]: Not sure what a similar record means in this context. Perhaps it is good to empohasize text?
```suggestion
### Order by textual record similarity
```

**Review by davidberenstein1957 — COMMENTED** [review]
Left some small comments.

File: argilla/docs/how_to_guides/annotate.md (hunk 1 above)
Comment (Amelie-V) [reply argilla/docs/how_to_guides/annotate.md:187]: Picky is fundamental! The global progress in dataset list is not exactly reflecting the records status we have into the dataset, Its more about providing a quick look on how its going the task. In any case with the task distribution V1 (Release 2.1) This Global progress will be probably simplified to Completed vs Left.

**Pull request closed without merging**